65595 Need to Void a Cash Receipt applied to Account only

sewa67 opened the pull request:
Please review this change and merge into development branch 
Files - Source/ar/b-cashl.w
Source/ar/b-onact1.w
Source/ar/d-arinvq.w
Source/ar/r-arve&p.w
Source/ar/r-cshe&p.w
Source/ar/r-mise&p.w
Source/ar/r-vcreg.w
Source/ar/v-cash.w
Source/ar/v-void.w
Source/ar/w-aronac.w
Source/sys/inc/create_nk1.p
Source/sys/ref/sys-ctrl.i
Source/windows/l-custact.w
Source/windows/l-ar-cash.w

sewa67 committed:
65595 Need to Void a Cash Receipt applied to Account only
Files - Source/ar/b-cashl.w
Source/ar/b-onact1.w
Source/ar/d-arinvq.w
Source/ar/r-arve&p.w
Source/ar/r-cshe&p.w
Source/ar/r-mise&p.w
Source/ar/r-vcreg.w
Source/ar/v-cash.w
Source/ar/v-void.w
Source/ar/w-aronac.w
Source/sys/inc/create_nk1.p
Source/sys/ref/sys-ctrl.i
Source/windows/l-custact.w
Source/windows/l-ar-cash.w

sewa67 committed:
65595 Need to Void a Cash Receipt applied to Account only
Files - Source/ar/r-vcreg.w

ron-advantzware reviewed (changes requested):
Procedure pCheckReceiptDatePeriod is identical in 3 of these programs
r-arve&p.w
r-mise&p.w
r-cshe&p.w

Please consolidate this procedure into an include and place the include in the above three instead.

sewa67 committed:
65595 Need to Void a Cash Receipt applied to Account only
Files - Source/ar/r-arve&p.w
Source/ar/r-cshe&p.w
Source/ar/r-mise&p.w
Source/ar/checkPeriod.i

ron-advantzware committed:
Merge branch 'develop' into feature/65595_Need_to_Void_a_Cash_Receipt_applied_to_Account_only
